Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "prometheus"
+version = "0.1.0"
+keywords = ["prometheus", "metrics"]
+
+[features]
+default = []
+dev = ["clippy"]
+
+[dependencies]
+protobuf = "1.0"
+quick-error = "0.2"
+clippy = {version = "*", optional = true}
```

Comment:
`*` dep-versions are usually discouraged and forbidden on [crates.io](https://github.com/rust-lang/rfcs/blob/master/text/1241-no-wildcard-deps.md).